preprod.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json

```diff
@@ -663,19 +663,11 @@
         "supportedValues": {
           "accessible": {
             "enabled": true,
-            "menuText": "Available"
+            "menuText": "Accessible"
           },
           "unaccessible": {
             "enabled": true,
             "menuText": "Not Accessible"
-          },
-          "pending": {
-            "enabled": true,
-            "menuText": "Pending"
-          },
-          "notAvailable": {
-            "enabled": true,
-            "menuText": "Not Available"
```

Comment:
Might be missing something but I don't see any indication of PXP-10660 getting solved with this PR. Can we link that JIRA issue with this PR?